### PR TITLE
Feature: add orientation markers tool

### PR DESCRIPTION
**Pull request by frolic06**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

* **What is the current behavior?** (You can also link to an open issue here)
No

* **What is the new behavior (if this is a feature change)?**
It adds a tool called OrientationMarkersTool, which displays orientation markers like in version 2.X

orientation markers

Some code:
`const orientationMarkers = cornerstoneTools['OrientationMarkersTool'];`
`cornerstoneTools.addToolForElement(element, orientationMarkers);`
`cornerstoneTools.setToolActiveForElement(this.element, 'OrientationMarkers', {});`

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

**Comment by dannyrb** [comment]
@frolic06 is this ready for review?

**Comment by frolic06** [comment]
Yes

**Comment by frolic06** [comment]
Hi @dannyrb , 
there is an issue related to this PR: #967

**Comment by frolic06** [comment]
@dannyrb did you have to review this PR ?
Can I help you in some way ?

File: src/index.js [hunk 1]
```diff
@@ -404,6 +413,8 @@ export {
   SaveAs,
   enableLogger,
   disableLogger,
+  calculateSUV,
+  convertToVector3,
```
Comment (dannyrb) [inline src/index.js]: These are `lib` exports:

- [convertToVector3](https://github.com/cornerstonejs/cornerstoneTools/blob/master/src/lib.js#L154)
- [calculateSUV](https://github.com/cornerstonejs/cornerstoneTools/blob/master/src/lib.js#L135)

You can import them like this:

```js
import { importInternal } from 'cornerstone-tools';

const convertToVector3 = importInternal('util/convertToVector3');
```

File: src/stackTools/stackPrefetch.js [hunk 2]
```diff
@@ -239,7 +239,8 @@ function prefetch(element) {
       requestType,
       preventCache,
       doneCallback,
-      failCallback
+      failCallback,
+      true
```
Comment (dannyrb) [inline src/stackTools/stackPrefetch.js]: While `addToBeginning` could potentially default to true, it feels outside the scope of this PR

File: src/tools/OrientationMarkersTool.test.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      external.cornerstone.getEnabledElement.mockReturnValue({ image: {} });
+      external.cornerstone.updateImage = jest.fn();
+
+      const instantiatedTool = new OrientationMarkersTool();
+
+      instantiatedTool.forceImageUpdate({});
+      expect(external.cornerstone.updateImage).toHaveBeenCalled();
+    });
+
+    it('should NOT call updateImage is image does not exists', () => {
```
Comment (dannyrb) [inline src/tools/OrientationMarkersTool.test.js]: is --> if (typo)

File: src/index.js (hunk 1 above)
Comment (frolic06) [reply src/index.js]: I wasn't aware of this function! I will remove this from the pull request.

File: src/stackTools/stackPrefetch.js (hunk 2 above)
Comment (frolic06) [reply src/stackTools/stackPrefetch.js]: I'm agree, I should have made another PR for this change.

File: src/tools/OrientationMarkersTool.test.js (hunk 3 above)
Comment (frolic06) [reply src/tools/OrientationMarkersTool.test.js]: ok !

File: src/stackTools/stackPrefetch.js (hunk 2 above)
Comment (frolic06) [reply src/stackTools/stackPrefetch.js]: Can I leave that or do you prefer a new PR ?

**Comment by dannyrb** [comment]
https://deploy-preview-1089--cornerstone-tools.netlify.com/netlify-example/overlay/index.html

image

**Comment by dannyrb** [comment]
Thanks for your hard work on this, @frolic06! ^_^
